.environment-scripts/stellar/install_bat.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -e
+
+INSTALL_DIR=$HOME/software/bats_dest/
+CLONE_PREFIX=$HOME/software/bats_clone/
```

Comment:
Could you propagate the `$INSTALL_PREFIX` and `$CLONE_PREFIX` in from the `setup_base_environment.sh` script via arguments?  The motivation is to make it so that it is easy to install multiple independent versions of the environment if necessary (e.g. if you want to run a set of experiments with one version of the code base and dependencies, and another set of experiments with another).

Reply:
Done